src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -54,7 +54,7 @@ public class RequestExecutor : IDisposable
 
         private static readonly GetStatisticsOperation BackwardCompatibilityFailureCheckOperation = new GetStatisticsOperation(debugTag: "failure=check");
         private static readonly DatabaseHealthCheckOperation FailureCheckOperation = new DatabaseHealthCheckOperation();
-        private ConcurrentSet<string> _useOldFailureCheckOperation;
+        private static ConcurrentSet<string> _useOldFailureCheckOperation;
```

Comment:
That isn't safe. What happens when we have two concurrent document stores to different servers?

Reply:
Different server with different Urls so the only downside is a little bit bigger dictionary and so a little bit more time to add/search, no?

Reply:
The value of the `string` is the URL, node the node id?

Reply:
What do you mean by "node id"? The tag?
The current implementation uses the URL includes the protocol (http/s).

Reply:
If the URL is there, fine by me